### DISCUSSION TITLE
Why uniswapV3FlashCallback() was not implemented?

### DISCUSSION THREAD

**Discussion by itsabinashb** [Jeiwan/uniswapv3-book#64] [Milestone 3. Cross-tick Swaps]
While adding Flash functionality we created `IUniswapV3FlashCallback.sol` interface and `uniswapV3FlashCallback()` inside it but we didn't implement the function anywhere in repository, why? If we don't implement the function it will not work, isn't it?

**Comment by F4r84D** [comment]
Create an interfaces folder and import them to pool contract —> IUniswapV3MintCallback.sol & IUniswapV3MintCallback.sol

```
//SPDX-License-Identifier: MIT

pragma solidity ^0.8.14;

interface IUniswapV3SwapCallback {
    function uniswapV3SwapCallback(int256 amount0, int256 amount1, bytes calldata data) external;
} 
```
And same for IUniswapV3MintCallback.sol
